docs/how-to-guides/speech-engine.rst

```diff
@@ -0,0 +1,63 @@
+Speech Engines
+==============
+
+VoiceBase offers three distinct proprietary speech engines, each tuned to address different use cases. The table below shows feature support by engine.
+
+**Europa - Premium Engine**
```

Comment:
Premium...because of $$$.  but Europa does not have voiceFeatures; required for some classifiers. 
Also, Europa does its own number formatting....it can't be disabled.